app/lib/support_ticket.rb

```diff
@@ -0,0 +1,58 @@
+require "gds_zendesk/client"
+require "gds_zendesk/dummy_client"
+
+class SupportTicket
+  def self.client
+    @client ||= if Rails.env.development?
+                  GDSZendesk::DummyClient.new(logger: Rails.logger)
+                else
+                  credentials = ZENDESK_CREDENTIALS.merge(logger: Rails.logger)
+                  GDSZendesk::Client.new credentials
+                end
+  end
+
+  def self.send(hash)
```

Comment:
Might be better to go with `.call` which seems the most common approach for a single static interface on a class.

You can also use Ruby 3.x syntax to make this just:

```
def self.send(..) = new(...).send
```
and then you don't need to keep args in sync or use two extra lines.